modules/Layout/Boilerplate/Boilerplate.module.scss

```diff
@@ -5,10 +5,11 @@
 }
 
 .columns {
-    display: block;
-
+    display: grid;
+    gap: $spacing-6;
```

Comment:
Although the value for the gap is the same as it was, due to differences between `flex` and `grid`, the visual gap is now slightly smaller than it used to be. I would bump the gap to `$spacing-7`;

cc @iamhugofelix

Reply:
Another note for mobile display: The `.about` container still has the margin applied to it, so now the vertical spacing is twice of what it used to be:
<img width="534" alt="image" src="https://user-images.githubusercontent.com/1641218/185582029-c50665c9-8a3e-4081-9181-80732d819bc5.png">

I believe it's still better to use regular `display: block` for mobile and tablet screens.

Reply:
@oleg-semyonov @iamhugofelix Thank you for your feedbacks.

@oleg-semyonov I agree that it is better to keep the `display: block` for mobile and tablet screens. The other possible take would have been to use `grid-template-areas` to display schematics between desktop and small screens as following: 

```
    display: grid;
    grid-template-areas:
    'aboutUs'
    'contacts';
    @include desktop-up {
        grid-template-areas: 'aboutUs aboutUs contacts';
    }
```

In this case, we would have to return to using multiple margins for elements in a container instead of one single `gap`, which would defeat the purpose of cleaning up the code. I think it would be better to keep the current version and close this PR. Let me know if you agree to close this PR.

Reply:
@d3b0nair Yeah, that makes sense. Feel free to close the PR if you think it's not justified to continue with it.